modules/cnv-restarting-vm-web.adoc

```diff
@@ -9,19 +9,29 @@ You can restart a running virtual machine from the web console.
 
 [IMPORTANT]
 ====
-Do not restart a virtual machine while it has a status of *Importing*. Restarting
-the virtual machine causes an error for it.
+To avoid errors, do not restart a virtual machine while it has a status of *Importing*.
 ====
 
 .Procedure
 
-. In the {CNVProductName} console, click *Workloads* -> *Virtual Machines*.
-. Restart the virtual machine from this screen, which makes it easier to perform
-actions on multiple virtual machines in the one screen, or from
-the *Virtual Machine Details* screen where you can view comprehensive details of
-the selected virtual machine:
-** Click the Options menu {kebab} at the end of virtual machine and select
-*Restart Virtual Machine*.
-** Click the virtual machine name to open the *Virtual Machine Details* screen
-and click *Actions* and select *Restart Virtual Machine*.
+. Click *Workloads* -> *Virtual Machines*.
+
+. Find the row that contains the virtual machine that you want to restart.
+
+. Restart the virtual machine:
```

Comment:
Bit of a nit but this step doesn't cover the action itself but finding the applicable menu.